spec/requests/swagger_spec.rb

```diff
@@ -0,0 +1,454 @@
+require 'rails_helper'
+
+RSpec.describe 'Swagger spec', type: :apivore, order: :defined do
```

Comment:
is it possible to break these out into separate spec files for each endpoint? Find it easier to read / understand them that way